docs/source/algorithms/LoadISISNexus-v2.rst

```diff
@@ -14,13 +14,118 @@ Loads a Nexus file created from an ISIS instrument.
 Data loaded from Nexus File
 ###########################
 
-The nexus file must have ``/raw_data_1`` as its main group and
+The nexus file must have ``raw_data_1`` as its main group and
 contain a ``/isis_vms_compat`` group to be loaded.
 
-The workspace data is loaded from ``/raw_data_1/Detector_1``. 
-Instrument information is loaded ``/raw_data_1/Instrument``, if available there and not overriden.
+The workspace data is loaded from ``raw_data_1/Detector_1``. 
+Instrument information is loaded ``raw_data_1/Instrument``, if available there and not overriden.
 Also the ``NSP1``, ``UDET``, ``SPEC``, ``HDR``, ``IRPB``, ``RRPB``, ``SPB`` and ``RSPB`` sections of
```

Comment:
This is a huge improvement towards knowing what LoadISISNexus does.
It would be helpful to have an idea what those acronyms mean (IRPB, RRPB, SPB, RSPB). I'm not sure if we can find that information or who could shed some light though.